examples/with-clerk/app/layout.tsx

```diff
@@ -0,0 +1,58 @@
+import '../styles/globals.css'
+import { ClerkProvider, SignedIn, SignedOut, UserButton } from '@clerk/nextjs'
+import { Inter } from 'next/font/google'
+import Image from 'next/image'
+import Script from 'next/script'
+import styles from '../styles/Header.module.css'
+import Link from 'next/link'
+
+const inter = Inter({ subsets: ['latin'] })
+
+export const metadata = {
+  title: 'Create Next App',
+  description: 'Generated by create next app',
+}
+
+const Header = () => (
+  <header className={styles.header}>
+    <div className={styles.left}>
+      <Link href="/" className={styles.logo}>
+        <Image src="/logo.svg" width="32" height="32" alt="Logo" />
+        <span className={styles.appName}>Your application</span>
+      </Link>
+    </div>
+    <div className={styles.right}>
+      <SignedOut>
+        <Link href="/sign-in">Sign in</Link>
+      </SignedOut>
+      <SignedIn>
+        <UserButton />
+      </SignedIn>
+    </div>
+  </header>
+)
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode
+}) {
+  return (
+    <html lang="en">
+      <head>
+        <link
```

Comment:
We shouldn't be using `link` tags directly here, it should use the metadata API.

For external CSS files, you should directly import the CSS file in the related component.

Reply:
Fixed, didn't realize any external stylesheets have to be colocated with the code